ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/queries/CommonQueries.scala

```diff
@@ -37,13 +36,13 @@ trait CommonQueries extends Queries {
       .as(
         (long("sequence_no")
           ~ get[Option[InputStream]]("entry_id")
-          ~ get[Option[Array[Byte]]]("envelope")
+          ~ get[Option[InputStream]]("envelope")
           ~ get[Option[Long]]("heartbeat_timestamp")).map {
           case index ~ Some(entryId) ~ Some(envelope) ~ None =>
             index -> LedgerEntry.LedgerRecord(
               Offset(Array(index)),
-              DamlLogEntryId.parseFrom(entryId),
-              envelope,
+              ByteString.readFrom(entryId),
+              ByteString.readFrom(envelope),
```

Comment:
I really like the use of `implicit def byteStringToStatement`. Can we do a similar thing for this (and the code in `selectStateValuesByKeys`)?

Reply:
Sure, made that change.